src/main/java/lotto/step2/domain/Lotto.java

```diff
@@ -0,0 +1,27 @@
+package lotto.step2.domain;
+
+import java.util.ArrayList;
+
+public class Lotto {
+    private ArrayList<LottoTicket> lotto;
```

Comment:
ArrayList 를 사용하신 이유가 궁금하네요?
```suggestion
    private List<LottoTicket> lotto;
```